.github/workflows/controller-image.yaml

```diff
@@ -137,37 +135,3 @@ jobs:
         id: make-catalog-push
         run: make catalog-push REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=${{ github.ref_name }}
       
-      - name: Print Image URL
-        run: echo "Image pushed to ${{ steps.push-to-quay.outputs.registry-paths }}" 
-
-  update-hcg-unstable:
```

Comment:
This hasnt passed in a long time and is referencing a old glbc repo so I'm persumming its not needed